capa/main.py

```diff
@@ -429,40 +429,38 @@ def is_running_standalone() -> bool:
     return hasattr(sys, "frozen") and hasattr(sys, "_MEIPASS")
 
 
-def get_default_root() -> str:
+def get_default_root() -> Path:
     """
-    get the file system path to the default resources directory.
-    under PyInstaller, this comes from _MEIPASS.
-    under source, this is the root directory of the project.
+    Get the file system path to the default resources directory.
+    Under PyInstaller, this comes from _MEIPASS.
+    Under source, this is the root directory of the project.
```

Comment:
unless you're going to change all the capitalization across this file, i'd recommend leaving the comments as-is for consistency (although grammatically incorrect).